gobblin-data-management/src/main/java/gobblin/data/management/copy/CopyableFileFilter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.data.management.copy;
+
+import java.util.Collection;
+
+import org.apache.hadoop.fs.FileSystem;
+
+
+/**
+ * A filter that is applied on all the {@link CopyableFile}s found by
+ * {@link CopyableDataset#getCopyableFiles(FileSystem, CopyConfiguration)}.
+ */
+public interface CopyableFileFilter {
+
+  public Collection<CopyableFile> filter(FileSystem sourceFs, FileSystem targetFs,
```

Comment:
Missing Javadoc that should document clearly what the filtering does and what's the expected output.